<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyNotesMaker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/img/icon.jpg</normaloff>:/img/img/icon.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="styleSheet">
       <string notr="true">#widget_4 {
    background-color: rgb(243, 243, 243);
    border-radius: 10px;
    border: 4px solid orange;
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="main">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="header" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="styleSheet">
                    <string notr="true">QLabel { 
	background-color: rgba(0, 0, 0, 0);
    border: none; 
	font: 700 14pt Segoe Script;
    color: rgb(255, 136, 0);
}
QLabel:hover {
    color: rgb(255, 255, 0);
}</string>
                   </property>
                   <property name="text">
                    <string>MyNotesMaker</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="topbtn">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset resource="res.qrc">
                        <normaloff>:/img/img/location_on_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/location_on_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="hidebtn">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset resource="res.qrc">
                        <normaloff>:/img/img/minimize_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/minimize_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="fullbtn">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset resource="res.qrc">
                        <normaloff>:/img/img/fullscreen_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/fullscreen_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="closebtn">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset resource="res.qrc">
                        <normaloff>:/img/img/close_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/close_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="main_2" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QTabWidget" name="tabWidget">
                   <property name="styleSheet">
                    <string notr="true">QTabWidget::tab-bar {
    alignment: center;
}

</string>
                   </property>
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="tab">
                    <attribute name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/img/img/description_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/description_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                    </attribute>
                    <attribute name="title">
                     <string>Document</string>
                    </attribute>
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QTextEdit" name="textEdit">
                       <property name="font">
                        <font>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="acceptRichText">
                        <bool>false</bool>
                       </property>
                       <property name="placeholderText">
                        <string>Start typing here...</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="tab_2">
                    <attribute name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/img/img/info_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/info_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                    </attribute>
                    <attribute name="title">
                     <string>Property</string>
                    </attribute>
                   </widget>
                   <widget class="QWidget" name="Save">
                    <attribute name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/img/img/save_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/save_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                    </attribute>
                    <attribute name="title">
                     <string>Save</string>
                    </attribute>
                   </widget>
                   <widget class="QWidget" name="tab_3">
                    <attribute name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/img/img/history_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/history_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                    </attribute>
                    <attribute name="title">
                     <string>History</string>
                    </attribute>
                   </widget>
                   <widget class="QWidget" name="ChatGPT">
                    <attribute name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/img/img/chat_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/chat_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                    </attribute>
                    <attribute name="title">
                     <string>ChatGPT</string>
                    </attribute>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="about">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="topMargin">
               <number>7</number>
              </property>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="backBtn">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/img/img/arrow_back_new_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/img/img/arrow_back_new_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="styleSheet">
                    <string notr="true">QLabel { 
	background-color: rgba(0, 0, 0, 0);
    border: none; 
	font: 700 14pt Segoe Script;
    color: rgb(255, 136, 0);
}
QLabel:hover {
    color: rgb(255, 255, 0);
}</string>
                   </property>
                   <property name="text">
                    <string>MyNotesMaker</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>225</width>
                   <height>703</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="widget_3" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <property name="leftMargin">
                      <number>5</number>
                     </property>
                     <property name="topMargin">
                      <number>5</number>
                     </property>
                     <property name="rightMargin">
                      <number>5</number>
                     </property>
                     <property name="bottomMargin">
                      <number>5</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="img">
                       <property name="minimumSize">
                        <size>
                         <width>210</width>
                         <height>200</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>210</width>
                         <height>200</height>
                        </size>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="pixmap">
                        <pixmap resource="res.qrc">:/img/img/icon.jpg</pixmap>
                       </property>
                       <property name="scaledContents">
                        <bool>true</bool>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                       <property name="wordWrap">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignTop">
                   <widget class="QLabel" name="label_2">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel { 
	font: 700 10pt Segoe Script;
}</string>
                    </property>
                    <property name="text">
                     <string>Програма MyNotesMaker - це зручний інструмент для створення, збереження та управління нотатками. Завдяки простому та інтуїтивно зрозумілому інтерфейсу користувача, MyNotesMaker дозволяє легко створювати нові нотатки, редагувати вже існуючі та швидко знаходити потрібну інформацію. </string>
                    </property>
                    <property name="scaledContents">
                     <bool>false</bool>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
